OutOfSchool/OutOfSchool.WebApi/Models/OperationWithObjectDto.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel.DataAnnotations;
+using OutOfSchool.Services.Enums;
+
+namespace OutOfSchool.WebApi.Models;
+
+public class OperationWithObjectDto
```

Comment:
does it really need to have all fields mapped from data access at the moment? i've only seen the usage of 2-3 so far